### PR TITLE
:sparkles: slackbot+github oauth+getPRs

### PR DESCRIPTION
**Pull request by pathiery**

### CODE REVIEW

**Comment by pathiery** [comment]
ayooo

**Comment by agagnon2** [comment]
allo

File: github/authenticateWithGithub.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
+const { createOAuthDeviceAuth } = require("@octokit/auth-oauth-device");
+const sendMessageToUser = require("../slackbot/sendMessageToUser");
+const { addUserTokenToRegistry } = require('../registry/registryInterface');
```
Comment (agagnon2) [inline github/authenticateWithGithub.js:3]: pas tres constant le mec

**Comment by agagnon2** [comment]
@pathiery  la constance mec

File: github/authenticateWithGithub.js (hunk 1 above)
Comment (pathiery) [reply github/authenticateWithGithub.js:3]: so what

Comment (pathiery) [reply github/authenticateWithGithub.js:3]: oui

Comment (pathiery) [reply github/authenticateWithGithub.js:3]: lol

Comment (pathiery) [reply github/authenticateWithGithub.js:3]: oui

Comment (pathiery) [reply github/authenticateWithGithub.js:3]: I'm not sure about that ...

Comment (pathiery) [reply github/authenticateWithGithub.js:3]: test

Comment (pathiery) [reply github/authenticateWithGithub.js:3]: I'm not too sure about that ...

Comment (pathiery) [reply github/authenticateWithGithub.js:3]: ❓

**Pull request closed without merging**